### PR TITLE
Exercise28

### PR DESCRIPTION
**Pull request by clucksupremacy**

### CODE REVIEW

File: exercise-28.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const prompt = require("prompt-sync")({ sigint: true });
+
+function countdown() {
+    let input = prompt('Enter a non-negative integer value (e.g. 0, 1, 2): ');
+
+    if (input < 0 || input % 1 != 0 || isNaN(input) === true) {
```
Comment (Aceeri) [inline exercise-28.js:6]: What's the `input % 1 != 0` check for here?

Comment (Aceeri) [reply exercise-28.js:6]: Oh right, floats nvm

File: exercise-28.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        console.log("Exit Program");
+        return input;
+    } else {
+        console.log("Counting down");
+        let intervalId = setInterval(() => {
+            if (input < 0 || input % 1 != 0 || isNaN(input) === true) {
```
Comment (Aceeri) [inline exercise-28.js]: I think it is safe to remove this, if the initial input is valid, then modifications here shouldn't make it invalid.

Comment (clucksupremacy) [reply exercise-28.js]: You're right, I simply forgot

**Pull request merged**